grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -758,13 +780,17 @@ public GrpcService build() {
             grpcService = new UnframedGrpcService(
                     grpcService, handlerRegistry,
                     unframedGrpcErrorHandler != null ? unframedGrpcErrorHandler
-                                                     : UnframedGrpcErrorHandler.of());
+                                                     : UnframedGrpcErrorHandler.of(),
+                    unframedGrpcStatusFunction != null ? unframedGrpcStatusFunction
+                                                       : UnframedGrpcStatusFunction.of());
```

Comment:
An `UnframedGrpcStatusFunction` return a null HTTP status. We can set the default `UnframedGrpcStatusFunction` as a fallback.
```ㅓava
// A `build()` method can be called multiple times, 
// we need a local variable here in order not to mutate this.unframedGrpcStatusFunction
UnframedGrpcStatusFunction unframedGrpcStatusFunction = firstNonNull(this.unframedGrpcStatusFunction, UnframedGrpcStatusFunction.of());
if (unframedGrpcStatusFunction != UnframedGrpcStatusFunction.of()) {
    unframedGrpcStatusFunction.orElse(UnframedGrpcStatusFunction.of());
}
```

Reply:
Updated but I didn't use `if` condition because I thought it is redundant by `firstNonNull` code above. 🤔